packages/renderer-preact/src/index.js

```diff
@@ -13,10 +13,8 @@ export default (Base = HTMLElement) =>
       };
     }
     renderer(root, call) {
-      this._preactDom = render(
-        call(),
-        root,
-        this._preactDom || root.children[0]
-      );
+      const children = root.children ||
```

Comment:
Do you think it would be safe to just do `childNodes[0]` in place of `children[0]`, instead of having to do the map? Also, do you mean to `filter` here instead?

Reply:
Ups, meant `filter`. I guess using `childNodes[0]` makes the most sense here. Will change it.